utils/utils.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Funcion para calcular el porcentaje de respuestas acertadas
+ */
+
+interface IOptions {
+  text: string;
+  options: Array<{text:string; correct: boolean}>;
+}
+
+export function grade(optionsQuestion: Record<string, boolean>[], answers: IOptions[]) {
```

Comment:
there is bug with this function try the next scenarios:


First scenario
Choose the first option in every question, and the result would be 2/2, but this is not expected because the are all wrong answers


Second scenario
Selects the first option to the first question, and then selects a right answer in the last one. The result is 2/2 when it should be 1/2


Also, take care of allQuestionsRight, it is not being restarted after end the first loop ( in other words, it not beginning with true and the beginning of the first loop)